### PR TITLE
F#src 2027 output static launch

### PR DESCRIPTION
**Pull request by dg-shadow**

### CODE REVIEW

File: sr_camera_pose_estimation/launch/calibration.launch [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <arg name="window_width" default="2000"/>
     <arg name="filtering" default="true"/>
     <arg name="continuous" default="true"/>
+    <arg name="output_static" default=""/>
```
Comment (biofotis) [inline sr_camera_pose_estimation/launch/calibration.launch]: no default value?

**Review by biofotis — COMMENTED** [review]
I think we should also update the readme file to explain this new option?

**Comment by dg-shadow** [comment]
default value is empty string

--

**Comment by dg-shadow** [comment]
fair point

--

**Review by biofotis — APPROVED** [review]

**Pull request merged**